src/main/kotlin/com/arkivanov/composnake/DefaultGame.kt

```diff
@@ -5,14 +5,12 @@ import androidx.compose.runtime.State
 import androidx.compose.runtime.mutableStateOf
 
 class DefaultGame : Game {
-
+    private val width = 16
```

Comment:
Since these two variables are not used outside of the `init` block, let's keep them inside.